### PR TITLE
updates and proxy support

### PR DESCRIPTION
**Pull request by thehesiod**
- add proxy support: based on
https://github.com/aio-libs/aiobotocore/pull/80 and corresponding
unittests
- remove limit/verify_ssl connector_args as they're now superseded by
botocore implementation

### CODE REVIEW

**Comment by jettify** [comment]
@thehesiod just curious, does this PR allows us to use moto server ? https://github.com/spulec/moto#stand-alone-server-mode

**Comment by jettify** [comment]
I tried to attach moto quickly sometime ago without lot of success...

**Comment by thehesiod** [comment]
probably, have a quick script?

**Comment by jettify** [comment]
Looks like working) that is super awesome !

**Comment by thehesiod** [comment]
aiobotocore is looking better and better!  When I have a LOT of free time I'll look at creating a PR in botocore to slowly integrate into them.

**Comment by jettify** [comment]
First:
```
pip install moto[server]
```
then spin server in separate console/process
```
moto_server s3
```
Then regular code but with additional parameter `endpoint_url`
```
    client = session.create_client('s3', region_name='us-west-2',
                                   endpoint_url='http://127.0.0.1:5000',
                                   aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
                                   aws_access_key_id=AWS_ACCESS_KEY_ID)
```
where AWS_SECRET_ACCESS_KEY and AWS_ACCESS_KEY_ID dummy strings

**Comment by thehesiod** [comment]
btw, I've been using `AWS_DEFAULT_PROFILE` environment variable to test with relevant entries in `~/.aws/credentials`

**Comment by jettify** [comment]
also I added `exoport HTTP_PROXY=http://127.0.0.1:5000`

**Comment by jettify** [comment]
this is good news, finally we will be able to add proper integration tests on travis ci

File: aiobotocore/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 from .session import get_session, AioSession
 
-__version__ = '0.1.1'
+__version__ = '0.1.2'
```
Comment (jettify) [inline aiobotocore/__init__.py]: Minor suggestion, lets tag not released version as `0.1.2a0` or something like that, so we can reason better about versions in dev and in wild

Comment (thehesiod) [inline aiobotocore/__init__.py]: re-versioned

**Pull request merged**